src/renderer/src/routes/(setup)/advanced-setup-step-3/+page.svelte

```diff
@@ -1,33 +1,44 @@
 <script lang="ts">
 	import { goto } from '$app/navigation';
+	import { Button, IconButton } from '$components';
+	import { ArrowRight, BackArrow } from '$icons';
 	import { i18n } from '$services';
-	import { BackArrow, ArrowRight } from '$icons';
-	import { IconButton, Button } from '$components';
 	import { appPassword } from '$stores';
 
+	const keystoreInitialized = window.sessionStorage.getItem('keystoreInitialized');
 </script>
 
-<div class="fixed top-0 left-0 right-0 app-region-drag flex items-center justify-between p-3 bg-[#DADADA12]">
+<div
+	class="app-region-drag fixed left-0 right-0 top-0 flex items-center justify-between bg-[#DADADA12] p-3"
```

Comment:
Do you maintain a global theme for launcher where general colors are stored? would be great if this color was defined there if it's referenced multiple times elsewhere

Reply:
To be honest, I didn't figure out how to do this ergonomically with tailwind...do you have a concrete way in mind on how you'd solve it? It doesn't seem to be possible to reference it as a variable in this tailwind class syntax, i.e. this does not seem to work: `bg-[var(--some-global-color-var)]`

Reply:
Yea, I have used tailwind in the past, it allows you to create a color palette in the `tailwind.config.js` file and you can reference those colors by name rather than by hex code, especially if the hex code is used multiple times

https://tailwindcss.com/docs/customizing-colors

Reply:
Oh cool, thanks! Pushed a commit with this addressed.